wrapper/Ada/tls_client.adb

```diff
@@ -40,8 +41,62 @@ package body Tls_Client with SPARK_Mode is
 
    subtype Byte_Type is WolfSSL.Byte_Type;
 
+   subtype chars_ptr is WolfSSL.chars_ptr;
+   subtype unsigned is WolfSSL.unsigned;
+
    package Natural_IO is new Ada.Text_IO.Integer_IO (Natural);
 
+   function PSK_Client_Callback
+     (Unused         : WolfSSL.WolfSSL_Type;
+      Hint           : chars_ptr;
+      Identity       : chars_ptr;
+      Id_Max_Length  : unsigned;
+      Key            : chars_ptr;
+      Key_Max_Length : unsigned) return unsigned
+   with Convention => C;
+
+   function PSK_Client_Callback
```

Comment:
Can you confirm this is in the tls_client example only and not part of the wolfSSL ADA wrapper? The PSK callback is meant to be customized for each customer.

Reply:
Yes, this is only an example for a (D)TLS client. In fact, I think these files should be put in an `examples/` directory and that would make it more obvious, and spare these lines:
https://github.com/wolfSSL/wolfssl/blob/78c4a04cac0647506807023b4d27b388ae4574c3/wrapper/Ada/wolfssl.gpr#L18-L23